hackathon_site/review/admin.py

```diff
@@ -15,8 +17,52 @@
 from review.views import MailerView
 
 
+class ReviewResource(resources.ModelResource):
+    class Meta:
+        model = Review
+        fields = (
+            "application__user__first_name",
+            "application__user__last_name",
+            "application__user__email",
+            "application__user__username",
+            "application__team__team_code",
+            "birthday",
```

Comment:
These are fields on the applications, not the reviews.

The fields should be:
- All the application user ones you have, those are good
- `reviewer__email`,
- `interest`
- `experience`
- `quality`
- `reviewer_comments`
- `status`
- `decision_sent_date`
- `created_at`
- `updated_at`

Set the export headers accordingly as well. "reviewer_email" (one underscore instead of two) etc.